source/common/config/delta_subscription_state.cc

```diff
@@ -76,10 +76,15 @@ void DeltaSubscriptionState::updateResourceInterest(
   }
 }
 
+void DeltaSubscriptionState::updateResourceInterestViaAliases(
+    const std::set<std::string>& updates_to_these_aliases) {
+  aliases_added_.insert(updates_to_these_aliases.begin(), updates_to_these_aliases.end());
```

Comment:
I think I'd want to see some more documentation/comments on how this module works with aliases, could you add some? CC @fredlas for this module review.

Reply:
One surface-level (since I don't know how it's supposed to work) question to double check you're doing what you intend: is it your intention that aliases are only ever added, not removed? Or, it looks like the actual mechanism is that these aliases just go into resource_names_subscribe; does that mean that to remove them, you are supposed to just do a normal unsubscribe of those names?